tests/agents/gherkin-specs/user_agent.feature

```diff
@@ -22,8 +22,8 @@ Feature: Agent Transport User agent Header
       | service_name    | <SERVICE_NAME>    |
       | service_version | <SERVICE_VERSION> |
     When the agent sends a request to APM server
-    Then the User-Agent header of the request matches regex '^apm-agent-[a-z]+/[^ ]* \(<ESCAPED_SERVICE_NAME> <ESCAPED_SERVICE_VERSION>\)'
+    Then the User-Agent header matches regex '^apm-agent-[a-z]+/[^ ]* \(<ESCAPED_SERVICE_NAME> <ESCAPED_SERVICE_VERSION>\)'
```

Comment:
I intended to change the other `Then the User-Agent header` statements in this spec to match this one, to align with the wording in the api_key.feature

https://github.com/elastic/apm/blob/3e8397358a1da8365a91ef9253d4a715574b36a9/tests/agents/gherkin-specs/api_key.feature#L8

Reply:
Ahh, this makes perfect sense! I'll adjust accordingly

Reply:
OK, adjusting to match that then